Components/Core/Spells/Equivalencies.lua

```diff
@@ -38,200 +38,229 @@ local _, pclass = UnitClass("Player")
+			 268050, -- Anchor of Binding (Shrine of the Storms dungeon)
+			 274389, -- Rat Traps (Freehold dungeon)
 		},
 		Slowed = {
-			   -116, -- Frostbolt
-			   -120, -- Cone of Cold
-			  -1715, -- Hamstring
-			   2120, -- Flamestrike
-			  -3409, -- Crippling Poison
-			  -3600, -- Earthbind
-			  -5116, -- Concussive Shot
-			   6343, -- Thunder Clap
-			  -7992, -- Slowing Poison
-			 -12323, -- Piercing Howl
-			 -12544, -- Frost Armor
-			 -13810, -- Ice Trap
-			 -15407, -- Mind Flay
-			 -31589, -- Slow
-			  35346, -- Warp Time
-			  44614, -- Flurry
-			  45524, -- Chains of Ice
+			   -116, -- Frostbolt (frost mage)
+			   -120, -- Cone of Cold (frost mage)
+			  -1715, -- Hamstring (arms warrior)
+			   2120, -- Flamestrike (fire mage)
+			  -3409, -- Crippling Poison (assassination rogue)
+			  -3600, -- Earthbind (shaman, general)
+			  -5116, -- Concussive Shot (beast mastery/marksman hunter)
+			   6343, -- Thunder Clap (protection warrior)
+			  -7992, -- Slowing Poison (NPC ability)
+			 -12323, -- Piercing Howl (fury warrior)
+			 -12544, -- Frost Armor (NPC ability only now?)
+			 -15407, -- Mind Flay (shadow priest)
+			 -31589, -- Slow (arcane mage)
+			  35346, -- Warp Time (NPC ability)
+			  44614, -- Flurry (frost mage)
+			  45524, -- Chains of Ice (death knight)
 			  50259, -- Dazed (Wild Charge, druid talent, cat form)
-			  50433, -- Ankle Crack
-			  51490, -- Thunderstorm
-			 -58180, -- Infected Wounds
-			  61391, -- Typhoon
-			 102793, -- Ursol's Vortex
-			 116095, -- Disable
-			 121253, -- Keg Smash
-			 123586, -- Flying Serpent Kick
-			 135299, -- Tar Trap
-			 147732, -- Frostbrand Attack
-			 157981, -- Blast Wave
-			 160065, -- Tendon Rip
-			 160067, -- Web Spray
-			 169733, -- Brutal Bash
-			 183218, -- Hand of Hindrance
-			 185763, -- Pistol Shot
-			 190780, -- Frost Breath (dk frost artifact ability)
-			 191397, -- Bestial Cunning
-			 194279, -- Caltrops
-			 194858, -- Dragonsfire Grenade
-			 195645, -- Wing Clip
-			-196840, -- Frost Shock
-			 198222, -- System Shock
-			 198813, -- Vengeful Retreat
-			 201787, -- Heavy-Handed Strikes (ww pvp talent)
-			 204263, -- Shining Force
-			 204843, -- Sigil of Chains
-			 205021, -- Ray of Frost (frost mage talent)
-			 205320, -- Strike of the Windlord
-			 205708, -- Chilled (frost mage effect)
-			 206755, -- Ranger's Net
+			  50433, -- Ankle Crack (hunter pet)
+			  51490, -- Thunderstorm (elemental shaman)
+			 -58180, -- Infected Wounds (feral druid)
+			  61391, -- Typhoon (druid, general)
+			 102793, -- Ursol's Vortex (resto druid)
+			 116095, -- Disable (windwalker monk)
+			 121253, -- Keg Smash (brewmaster monk)
+			 123586, -- Flying Serpent Kick (windwalker monk)
+			 135299, -- Tar Trap (survival hunter)
+			 147732, -- Frostbrand Attack (enhancement shaman)
+			 157981, -- Blast Wave (fire mage)
+			 160065, -- Tendon Rip (NPC ability)
+			 160067, -- Web Spray (pet ability)
+			 169733, -- Brutal Bash (NPC ability?)
```

Comment:
Doesn't appear to be a slow anymore. I believe this used to be a secondary effect of a warrior or maybe druid ability.